tc/core/tc2halide.cc

```diff
@@ -62,10 +62,13 @@ Type translateScalarType(int tcType) {
   }
 }
 
+// translate the TC def input params to corresponding Halide components.
+// params, inputs will be populated here
```

Comment:
Please start sentences with a capital and close them with a period.
This also applies to other comments added below.